events.go

```diff
@@ -186,7 +210,13 @@ type CommitCommentEvent struct {
 			DeletedFile bool   `json:"deleted_file"`
 		} `json:"st_diff"`
 	} `json:"object_attributes"`
-	Commit *Commit `json:"commit"`
+	Commit *struct {
+		ID        string     `json:"id"`
+		Message   string     `json:"message"`
+		Timestamp *time.Time `json:"timestamp"`
+		URL       string     `json:"url"`
+		Author    *Author    `json:"author"`
```

Comment:
Is using the `Author` struct good here? As you didn't use that in the other locations?